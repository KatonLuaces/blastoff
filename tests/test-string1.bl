A = 'BLAS';
print(toString(A));