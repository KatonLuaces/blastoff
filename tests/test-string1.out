66
76
65
83
